libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -4878,6 +4878,39 @@ MAV_RESULT GCS_MAVLINK::handle_command_request_autopilot_capabilities(const mavl
 }
 #endif
 
+#if AP_AHRS_ENABLED
+MAV_RESULT GCS_MAVLINK::handle_AHRS_message(const mavlink_command_int_t &packet)
+{
+    switch (packet.command) {
+        case MAV_CMD_AHRS_START:
+        {
+            AP::ahrs().set_data_sending_state(/*enabled*/ true);
```

Comment:
I know it's ON/OFF, but the fact you need a commen there may be a good reason we can do 

```c++
enum class GNSS_SENDING_STATE {
  ENABLED,
  DISABLED
};
```

Reply:
Yeah, I was thinking about that too.
If you don't mind, I'll start a separate next request with the "enum class" using

Reply:
Done